api/nodes/views.py

```diff
@@ -360,7 +364,10 @@ class NodeDetail(generics.RetrieveUpdateDestroyAPIView, NodeMixin):
 
     # overrides RetrieveUpdateDestroyAPIView
     def get_object(self):
-        return self.get_node()
+        node = self.get_node()
+        if node.is_registration:
```

Comment:
If they aren't part of the collection, then they return a 404 if you try to access them through that collection, despite how similar they seem.